lib/board.rb

```diff
@@ -0,0 +1,53 @@
+class Board
+
+  attr_accessor :cells
+
+  def initialize
+    @cells = Array.new(9, " ")
```

Comment:
Why not just call `reset!` here and use that logic instead of repeating what "resetting" a game does?

Try to rely on your abstractions like methods instead of literal logic.